compass/ocean/tests/hurricane/forward/__init__.py

```diff
@@ -34,15 +35,24 @@ def __init__(self, test_group, mesh, storm, init):
 
         init : compass.ocean.tests.hurricane.init.Init
             The test case that produces the initial condition for this run
+
+        use_lts : bool
+            Whether local time-stepping is to be used
         """
         mesh_name = mesh.mesh_name
+
+        if use_lts:
+            storm = storm + '_lts'
         subdir = os.path.join(mesh_name, storm)
         super().__init__(test_group=test_group,
                          subdir=subdir,
                          name=storm)
```

Comment:
These are both stylistic changes but I think it makes more sense to append `_lts` to the step name than to the `storm` variable.  I also have a strong preference for f-strings over string addition.

```suggestion

        if use_lts:
            name = f`{storm}_lts'
        else:
            name = storm
        subdir = os.path.join(mesh_name, name)
        super().__init__(test_group=test_group,
                         subdir=subdir,
                         name=name)
```

Reply:
As a results of this change, you will pass `storm = 'sandy'` to Analysis, but I think that will be cleaner anyway because you can revert several changes in that step with this change.  I don't think it's needed but you might have to pass `use_lts` to that step.